dags/openshift_nightlies/manifest.yaml

```diff
@@ -75,6 +75,10 @@ platforms:
       config: 
         install: acs.json
         benchmarks: acs.json
+    - name: chaos
+      config:
```

Comment:
if you want to create 3 dags one for aws,azure,gcp then you need to do it like how we've done it in the cloud platform, have a providers section and then the variants : 
versions: ["4.10"]
  providers: ["aws", "gcp", "azure"]
    variants: 
    - name: chaos
      config: 
        install: chaos.json
        benchmarks: chaos.json

Reply:
they don't need to do this. They just need to make a variant in the cloud platform like acs did